src/main/webapp/fellowship/index.html

```diff
@@ -1,108 +1,141 @@
-                            <p class="small text-dark">This certificate is awarded to</p>
-                            <h3 class="text-muted">{{name}}</h3>
-                            <p class="text-dark">
-                                The  Fellowship program is an initiative to recognise individuals who have actively
-                                contributed to the delivery of flagship programs of the Sustainable Education Foundation
-                            </p>
-                            <p class="font-italic text-muted small">
-                                DIGITALLY SIGNED FOR AND ON BEHALF OF THE SUSTAINABLE EDUCATION FOUNDATION
-                            </p>
-                        </div>
-                        <div class="mt-5">
-                            <p class="small">
-                                Date Issued: <span class="text-dark">{{createdAt}}</span><br>
-                                Certificate Id: <span class="text-dark">{{id}}</span>
-                            </p>
-                            <img src="../assets/img/brand/logo.png" class="img-fluid w-25">
-                        </div>
-                    </div>
-                </div>
+      <div class="shadow bg-white m-5">
+        <div class="container">
+          <div class="row p-md-5 p-3 position-relative">
+            <img
+              class="position-absolute h-100 d-none d-md-block certificate-bg-image"
+              src="images/certificate_background.svg"
+            />
+            <div class="col-md-8">
+              <h2 class="text-dark font-weight-900">
+                Certificate of Fellowship
+              </h2>
+              <div class="mt-md-4">
+                <p class="small text-dark">This certificate is awarded to</p>
+                <h3 class="text-muted">{{name}}</h3>
+                <p class="text-dark">
+                  The Fellowship program is an initiative to recognise
+                  individuals who have actively contributed to the delivery of
+                  flagship programs of the Sustainable Education Foundation
+                </p>
+                <p class="font-italic text-muted small">
+                  DIGITALLY SIGNED FOR AND ON BEHALF OF THE SUSTAINABLE
+                  EDUCATION FOUNDATION
+                </p>
+              </div>
+              <div class="mt-5">
+                <p class="small">
+                  Date Issued: <span class="text-dark">{{createdAt}}</span
+                  ><br />
+                  Certificate Id: <span class="text-dark">{{id}}</span>
+                </p>
+                <img
+                  src="../assets/img/brand/logo.png"
+                  class="img-fluid w-25"
+                />
+              </div>
             </div>
+          </div>
         </div>
+      </div>
     </div>
-</script>
-<script>
+  </script>
+  <script type="text/html" id="404-template">
+    <h3>Error: 404 Page not Found</h3>
+  </script>
```

Comment:
It's better if we could wrap this `h3` in some parent divs and add classes `container`, `row`, `col text-center` classes.

Reply:
I added the above changes